### PR TITLE
Ux 569 update readme

### PR DESCRIPTION
**Pull request by oscarkwan**
### 🛠 Purpose

---

Update readme with better content and more concise information.

View here for the readme file
https://github.com/acl-services/paprika/tree/UX-569-update-readme

### ✏️ Notes to Reviewer

---

### 🖥 Screenshots

---
 
 ### 📙 Storybook 
 <a href='http://storybooks.highbond-s3.com/paprika/UX-569-update-readme' target="_blank" rel="noopener">http://storybooks.highbond-s3.com/paprika/UX-569-update-readme</a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,89 +1,76 @@
...
+<img alt="GitHub issues" src="https://img.shields.io/github/issues/acl-services/paprika">
+<img alt="GitHub" src="https://img.shields.io/github/license/acl-services/paprika">
+</div>
+<br>
+
+Paprika is a collection of reusable components that are versioned and downloaded individually which is built and maintained by [Galvanize](https://www.wegalvanize.com).
+
+[Npm](https://www.npmjs.com/org/paprika) |
+[Storybook](https://paprika.highbond.com) | 
+[Airtable](https://airtable.com/tblKm8hk9hxOhSjqZ/viwWfRyRdodyS2xVv?blocks=hide)
```
Comment (mikrotron) [inline README.md]: This link is public, no login required!
https://airtable.com/shrkJwkvtbgc3FT22

File: README.md [hunk 2]
```diff
@@ -1,89 +1,94 @@
...
 
-All paprika components can be installed as:
+### Prerequisites
 
-```sh
-  npm install @paprika/ComponentToConsume
-```
-
-or
+Paprika has a `peerDependecy` of `styled-components` so make sure to install it as a dependency in your project.
```
Comment (alexzherdev) [inline README.md]:
```suggestion
Paprika has a `peerDependency` on `styled-components` so make sure to install it as a dependency in your project.
```
I think "on" is more commonly used

File: README.md [hunk 3]
```diff
@@ -1,89 +1,94 @@
...
-
-## Contributors guide.
+- Node v10.0.0+
+- Yarn v1.12.3+
 
-### Basic installation
+### Example of installing a component
 
-Follow the next steps to start coding on paprika
+The following is an example of the button component, but all other components will be installed in a similiar way.
```
Comment (alexzherdev) [inline README.md]:
```suggestion
The following is an example of the button component, but all other components will be installed in a similar way.
```

File: README.md (hunk 2 above)
Comment (oscarkwan) [reply README.md]: i can always count on u 😍

File: README.md [hunk 4]
```diff
@@ -1,89 +1,94 @@
...
+import React from "react";
+import Button from "@paprika/button";
+
+export default <Button>Hello</Button>;
 ```
 
-That's all you are ready to go from here you can:
+## Contributors guide
+
+We welcome anyone to contribute to paprika by submitting a pull request, filing a bug report or asking for a feature request. For more information please check out our [contributing guidelines](https://github.com/acl-services/paprika/wiki/Contributing-Guidelines) wiki page.
```
Comment (alexzherdev) [inline README.md]: A request is already an ask. Maybe "making a feature request"?

File: README.md [hunk 5]
```diff
@@ -1,89 +1,94 @@
...
-2.- Link packages with Lerna
+2. In your project:
 
-```sh
-$ npx lerna bootstrap
+```js
+import React from "react";
+import Button from "@paprika/button";
+
+export default <Button>Hello</Button>;
```
Comment (alexzherdev) [inline README.md]: This example makes no sense 😄 Let's at least say
```suggestion
export default () => <Button>Hello</Button>;
```

File: README.md [hunk 6]
```diff
@@ -1,89 +1,94 @@
 ![Paprika logo](https://user-images.githubusercontent.com/10501940/52080175-07327400-254c-11e9-9748-7a00f93a13a8.png)
 
-# Paprika
+<h1 align="center">Paprika</h1>
+<p align="center">Paprika is a react component library of the <a href="https://design.wegalvanize.com">Starling Design System</a>.</p>
```
Comment (alexzherdev) [inline README.md]:
```suggestion
<p align="center">Paprika is the React component library of the <a href="https://design.wegalvanize.com">Starling Design System</a>.</p>
```
not sure about the article (it's not some library, it's _the_ library? )

File: README.md [hunk 7]
```diff
@@ -1,89 +1,94 @@
...
+<img alt="react version" src="https://img.shields.io/badge/react-v16.8.2-green.svg">
+<img alt="node version" src="https://img.shields.io/badge/node-v10.0.0%2B-brightgreen.svg">
+<img alt="yarn version" src="https://img.shields.io/badge/yarn-v1.12.3%2B-yellowgreen.svg">
+<img alt="GitHub issues" src="https://img.shields.io/github/issues/acl-services/paprika">
+<img alt="GitHub" src="https://img.shields.io/github/license/acl-services/paprika">
+</div>
+<br>
 
-## Getting started
+Paprika is a collection of reusable components that are versioned and downloaded individually which is built and maintained by [Galvanize](https://www.wegalvanize.com).
```
Comment (alexzherdev) [inline README.md]: How about
```suggestion
Paprika is a collection of individually versioned and distributed reusable components built and maintained by [Galvanize](https://www.wegalvanize.com).
```

File: README.md (hunk 5 above)
Comment (oscarkwan) [reply README.md]: This is what happens when doing documentation while sick looool

File: README.md (hunk 6 above)
Comment (oscarkwan) [reply README.md]: What about just
`React component library of the Starling Design System`

Comment (oscarkwan) [reply README.md]: We say Paprika above anyways

Comment (alexzherdev) [reply README.md]: works for me 👍

**Comment by oscarkwan** [comment]
@alexzherdev @nahumzs 

Updated :) thanks for reviewing!

**Comment by johnpeele** [comment]
Perhaps we should capitalize Paprika and Storybook everywhere?

**Pull request merged**